custom/icds_reports/templates/icds_reports/dashboard.html

```diff
@@ -117,6 +117,17 @@ <h2 class="dashboard-title">
                               <nav class="navbar-menus fullmenu" id="hq-full-menu" role="navigation">
                                   <div class="nav-settings-bar pull-right">
                                       <ul class="nav navbar-nav" role="menu">
+                                          <li class="nav-menu-icon release-notes" ng-if="haveAccessToFeatures">
+                                              <span class="pointer">
+                                                  <a href="{% url 'download_release_notes' domain %}" download>
+                                                      <img class="release-notes-icon" src="{% static 'img/release_notes.png' %}"
+                                                          uib-popover-html="'Download release notes'"
+                                                          popover-placement="left"
+                                                          popover-trigger="'mouseenter'" >
+                                                      <span class="new-alert" ng-if="!isOlderThanAWeek">New!</span>
```

Comment:
did you test this `isOlderThanAWeek`? its making a get request which could take time to get a response. What happens while the request being resolved I think like sending get request is a async task no? how is this handling that

Reply:
I have added debug point and delayed the response. Because of data-binding, as soon as getting the response, template got updated

Reply:
👍